DigitalLearningSolutions.Web/Helpers/ControllerHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System;
+
+    public static class ControllerHelper
+    {
+        public static string GetControllerAspName(Type controllerType)

```

Comment:
Is `AspName` a thing? If not, the risk with calling it `AspName` is that readers will assume it is a thing they don't understand. To make it slightly less intimidating, perhaps `GetControllerNameForAspHttpMethods`?